### PR TITLE
some visualisations and edits

### PR DESCRIPTION
**Pull request by hydrogenbond007**

### CODE REVIEW

**Comment by hydrogenbond007** [comment]
cc @aakashthumaty

File: src/wormhole/1_whatIsWormhole.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # What is Wormhole?
 
-Wormhole V1 was introduced in 2020 by Certus One and was initially conceived as a traditional token bridge between Ethereum and Solana. It served as the first bridge on Solana and was responsible for bootstrapping a large amount of the liquidity in the early Solana and Serum ecosystems.
+Wormhole V1 was introduced in 2020 by Certus One and was initially conceived as a traditional token bridge between Ethereum and Solana. It served as the first bridge on Solana providing corss-chain swaps + wrapped assets and was responsible for bootstrapping a large amount of the liquidity in the early Solana and Serum ecosystems.
```
Comment (barnjamin) [inline src/wormhole/1_whatIsWormhole.md:3]: typo s/corss/cross

File: src/technical/solana/overview.md [hunk 2]
```diff
@@ -7,3 +7,6 @@ The deployed contract addresses can be found on the [Contracts page](../../refer
 Solana is a rust-based programming environment. You'll likely find the [Wormhole Rust SDK](https://github.com/wormhole-foundation/wormhole/tree/main/sdk/rust) useful.
 
 The Solana integration is also fully supported by the [Wormhole Typescript SDK](https://github.com/wormhole-foundation/wormhole/tree/main/sdk/js).
+
+To get started on Solan devloplment you [can start here ](https://soldev.app/course)
```
Comment (barnjamin) [inline src/technical/solana/overview.md:11]: extra space, typo on development

File: src/dapps/5_advantages.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 Here are a few xDapp features that are making an impact across blockchain technologies:
 
-- **Expanded User Base** - Rather than being limited to the users of one blockchain, any user on any blockchain in the ecosystem can interact with an xDapp.
+- **Expanded User Base** - Rather than being limited to the users of one blockchain, any user on any blockchain in the ecosystem can interact with an xDapp which Wormhole supports.
```
Comment (barnjamin) [inline src/dapps/5_advantages.md:5]: no need to have a "wormhole supported" dapp? if anything id be the other way around, an xdapp that supports wormhole

File: src/dapps/2_crossChainInteroperability.md [hunk 4]
```diff
@@ -10,19 +10,21 @@ To solve the centralization problems with CEXs, decentralized exchanges (DEXs) w
 
 There are some other essential things you should know about bridges before going further:
 
-- Bridges are capable of being decentralized in theory, but are often quite centralized in practice.
-- Bridges are currently the only way to hold a token on a chain other than its 'native' chain. If you're holding ETH on a chain other than Ethereum, it is, by definition, a wrapped token.
+- Bridges are capable of being decentralized in theory, but every Bridge has its own trust and security assumptions as its impossibl to make a truelly trustless bridge.
```
Comment (barnjamin) [inline src/dapps/2_crossChainInteroperability.md:13]: typos impossible and truly

File: src/technical/solana/core.md [hunk 5]
```diff
@@ -0,0 +1 @@
+Solana uses Berkely packet filter as its execution environment aka Virtual Machine to execute arbitrary smart contract logic in a secure way.
```
[the remark this answers is not in the document]
Comment (hydrogenbond007) [reply src/technical/solana/core.md:1]: there was already a file on every chain, planning to do a small write on all of their main tech specs soon in the new docs

**Comment by evan-gray** [comment]
This repo is being archived in favor of docs.wormhole.com

**Pull request closed without merging**